Documentation/Helm-Charts/ceph-cluster-chart.gotmpl.md

```diff
@@ -73,52 +73,58 @@ with the specs from those examples.
 
 The `cephBlockPools` array in the values file will define a list of CephBlockPool as described in the table below.
 
-| Parameter | Description | Default |
-| --------- | ----------- | ------- |
-| `name` | The name of the CephBlockPool | `ceph-blockpool` |
-| `spec` | The CephBlockPool spec, see the [CephBlockPool](../CRDs/Block-Storage/ceph-block-pool-crd.md#spec) documentation. | `{}` |
-| `storageClass.enabled` | Whether a storage class is deployed alongside the CephBlockPool | `true` |
-| `storageClass.isDefault` | Whether the storage class will be the default storage class for PVCs. See [PersistentVolumeClaim documentation](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) for details. | `true` |
-| `storageClass.name` | The name of the storage class | `ceph-block` |
-| `storageClass.parameters` | See [Block Storage](../Storage-Configuration/Block-Storage-RBD/block-storage.md) documentation or the helm values.yaml for suitable values | see values.yaml |
-| `storageClass.reclaimPolicy` | The default [Reclaim Policy](https://kubernetes.io/docs/concepts/storage/storage-classes/#reclaim-policy) to apply to PVCs created with this storage class. | `Delete` |
-| `storageClass.allowVolumeExpansion` | Whether [volume expansion](https://kubernetes.io/docs/concepts/storage/storage-classes/#allow-volume-expansion) is allowed by default. | `true` |
-| `storageClass.mountOptions` | Specifies the mount options for storageClass | `[]` |
-| `storageClass.allowedTopologies` | Specifies the [allowedTopologies](https://kubernetes.io/docs/concepts/storage/storage-classes/#allowed-topologies) for storageClass | `[]` |
+| Parameter                           | Description                                                                                                                                                                                                          | Default          |
```

Comment:
Did your editor automatically change the table alignment whitespace? We would like to keep consistency in the table formatting, so would be preferred if the whitespace wasn't updated like this.

Reply:
I adjusted the table formatting now to be consistent